recipe/meta.yaml

```diff
@@ -19,10 +19,13 @@ requirements:
     - pip
     - python >=3.6
     - setuptools
+    - scipy ==1.7.3
+    - numpy =1.22
```

Comment:
Why do you need them here? They don't seem to be build requirements.

Reply:
`{{ PYTHON }} -m pip install . -vv` would pull them in anyway, might as well install them with conda, or use `--no-deps` during the installation.

Reply:
`--no-deps` is set via environment variables. The existing set of dependencies should be sufficient.